news/540.api_change

```diff
@@ -0,0 +1 @@
+A custom resolver interpolation whose output is a list or dictionary is now automatically converted into a ListConfig or DictConfig.
```

Comment:
Sure, would be easy to add. Is this specific functionality related to https://github.com/facebookresearch/hydra/issues/1389?

I wouldn't call it `dict_as_list` though, because in Python `list(some_dict)` returns the list of keys. So maybe `dict_values`? (could even be `oc.dict.values` if we want to start a namespace hierarchy)

Reply:
yeah, I like dict_values better.
oc.dict.values is interesting, do you think we will need more dict functions?

Reply:
lets do keys and values.
we can add items once we have a real use case.

Reply:
> lets do keys and values.
> we can add items once we have a real use case.

Added to the list in #535 => resolving